os/fs/smartfs/smartfs.h

```diff
@@ -409,6 +409,10 @@ void smartfs_setbuffer(struct smart_read_write_s *rw, uint16_t logsector, uint16
 
 void smartfs_set_entry_flags(struct smartfs_entry_s *new_entry, mode_t mode, uint16_t type);
 
+int smartfs_sync_internal(struct smartfs_mountpt_s *fs, struct smartfs_ofile_s *sf);
+
```

Comment:
can you remove this line?

Reply:
Kept same for all function declarations,
Will keep the line spacing for consistency